src/@types/react-diff-view/index.d.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint amo/only-tsx-files: 0, @typescript-eslint/no-unused-vars: 0 */
```

Comment:
Maybe we should rename it to `react-diff-view/index.d.tsx` if that's allowed?

Reply:
I tried but it did not work.. Let's try again, otherwise I'll fix that in the rule (later).

Reply:
It does not work, `tsc` does not recognize the module anymore.

Reply:
see: https://github.com/mozilla/eslint-plugin-amo/issues/17

Reply:
Ah, ok, I thought `.d.ts` might be special